docs/content/docs/dev/python/table/table_environment.md

```diff
@@ -199,6 +199,61 @@ These APIs are used to create/remove Table API/SQL Tables and write queries:
         {{< pythondoc file="pyflink.table.html#pyflink.table.TableEnvironment.drop_table" name="link">}}
       </td>
     </tr>
+    <tr>
+      <td>
+        <strong>drop_temporary_model(view_path, ignore_if_not_exists=True)</strong>
+      </td>
+      <td>
+        Drops a temporary model registered in the given path.
+      </td>
+      <td class="text-center">
+        {{< pythondoc file="pyflink.table.html#pyflink.table.TableEnvironment.drop_temporary_model" name="link">}}
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <strong>create_model(model_path, model_descriptor, ignore_if_exists=False)</strong>
+      </td>
+      <td>
+        Registers a `Model` object as a model similar to SQL Models. 
+      </td>
+      <td class="text-center">
+        {{< pythondoc file="pyflink.table.html#pyflink.table.TableEnvironment.create_model" name="link">}}
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <strong>create_temporary_model(model_path, model_descriptor, ignore_if_exists=False)</strong>
+      </td>
+      <td>
+        Registers a `Model` object as a temporary model similar to SQL temporary Models. 
```

Comment:
I suggest defining what a temporary model is and how it differs from a SQL temporary model.